lib/Doctrine/ORM/ORMException.php

```diff
@@ -4,221 +4,9 @@
 
 namespace Doctrine\ORM;
 
-use Doctrine\Common\Cache\Cache as CacheDriver;
-use Exception;
-use function get_class;
-use function implode;
-use function sprintf;
-
 /**
  * Base exception class for all ORM exceptions.
  */
-class ORMException extends Exception
+interface ORMException extends \Throwable
```

Comment:
Since we target master, can we maybe get rid of this polyfill and move it to its proper place (`Doctrine\ORM\Exception`)?

Reply:
Sure, why not?